core/src/main/scala/org/bitcoins/core/protocol/dlc/compute/DLCAdaptorPointComputer.scala

```diff
@@ -77,33 +119,59 @@ object DLCAdaptorPointComputer {
         }
       }
 
+    lazy val additionTries = preComputeTable.map { table =>
+      AdditionTrieNode(table, pointOpt = Some(SecpPointInfinity))
+    }
+
     val oraclesAndOutcomes = contractInfo.allOutcomes.map(_.oraclesAndOutcomes)
 
     oraclesAndOutcomes.map { oracleAndOutcome =>
       // For the given oracleAndOutcome, look up the point in the preComputeTable
-      val subSigPoints = oracleAndOutcome.flatMap { case (info, outcome) =>
-        val oracleIndex =
-          contractInfo.oracleInfo.singleOracleInfos.indexOf(info)
-        val outcomeIndices = outcome match {
-          case outcome: EnumOutcome =>
-            Vector(
-              contractInfo.contractDescriptor
-                .asInstanceOf[EnumContractDescriptor]
-                .keys
-                .indexOf(outcome)
-            )
-          case UnsignedNumericOutcome(digits) => digits
-          case _: SignedNumericOutcome =>
-            throw new UnsupportedOperationException(
-              "Signed numeric outcomes not supported!")
-        }
+      val subSigPoints = CryptoUtil.cryptoContext match {
+        case CryptoContext.LibSecp256k1 =>
+          oracleAndOutcome.flatMap { case (info, outcome) =>
+            val oracleIndex =
+              contractInfo.oracleInfo.singleOracleInfos.indexOf(info)
+            val outcomeIndices = outcome match {
+              case outcome: EnumOutcome =>
+                Vector(
+                  contractInfo.contractDescriptor
+                    .asInstanceOf[EnumContractDescriptor]
+                    .keys
+                    .indexOf(outcome)
+                )
+              case UnsignedNumericOutcome(digits) => digits
```

Comment:
this doesn't seem correct, should we be getting the index here, not the actual digit?

Reply:
The digit is the index of the outcome (0 is 0, 1 is 1, etc.)

Reply:
ah duh